libs/spandrel/spandrel/architectures/RealCUGAN/arch/upcunet_v3.py

```diff
@@ -419,3 +419,55 @@ def forward(self, x: Tensor, alpha: float = 1):
             x = (x - 0.15) / 0.7
 
         return x
+
+
+class pixel_unshuffle(nn.Module):
```

Comment:
I know it's just because it was in the original code, but why on earth wouldn't they just use torch.nn.pixelshuffle??

Reply:
In fact, this arch could probably be sped up by just swapping that out

Reply:
This might have been a compatibility thing to support older pytorch versions, idk. I just tested it with `nn.PixelUnshuffle` and the output is the same. So I agree that we should use the torch version.